your-toolbelt.md

```diff
@@ -0,0 +1,47 @@
+### Your toolbelt
+
+
+#### Exercise: Uses and understands unit, functional and integration tests
+
+1. You've now written two types of tests. Name them, and explain the differences.
+The test's writen were the Unit Tests and the Integration Tests. The Unit Tests are to teste a small piece of code, mainly to test the models (the views and the controllers aren't so much tested, in case of the views because it can change in a short period of time) Ideally, there will be, at least, one test for each method. The Integration Tests are maded to test differentes pieces of code of the system and garantee the they work together. In the simple sinatra sitiuation, the integration tests were to navegate in the routes, fill a specific field, click a button and expect the page to have a specific text.
+
+2. The other tests we haven't made are the Functional Tests, that are the tests to the controllers. The controllers receive incomming web requeste and eventually respond with a view.
```

Comment:
Do not use the word `made`, say `written`.

Controllers get `HTTP` requests, web requests is understandable, but not correct. Also, they will respond with an `HTTP` response, which can have the `Content-Type` of `text/html` and the content may be an `HTML` page, or it may be a `JSON`, `XML`, etc, response, so be careful when you say it responds with a view.